compile.sh

```diff
@@ -8,6 +8,10 @@ declare -a SRC
 SRC=(htable rng utils pairs store acc rotate query)
 CC=${OTHERC:-gcc}
 
+LEAKCHECK_ENABLED=${LEAKCHECK_ENABLED:-}
+LEAKCHECK=${OTHERMEMCHECK:-valgrind}
+LEAKCHECK_ARGS="--leak-check=full --trace-children=yes --error-exitcode=1"
```

Comment:
I usually also add `--track-origins=yes` as it can tell you where something was allocated in the case of a leak.